app/views/organisations/index.html.erb

```diff
@@ -1,4 +1,11 @@
-<h1 class="govuk-heading-xl">Organisations</h1>
+<h1 class="govuk-heading-xl">
+  <%= current_page?(action: "index") ? "Organisations" : "Organisations without active users" %>
+</h1>
+
+<p class="govuk-body">
+  <%= link_to "All organisations", { action: "index" } unless current_page?(action: "index") %>
```

Comment:
I think the content for this reads better if the links contain a verb, so `View all organisations` and `View organisations without active users`.

minor

Reply:
Yep, good feedback. Fixed and force-pushed 77e35ab